### PR TITLE
Add some initial text for documentation

### PR DESCRIPTION
**Pull request by AlanCoding**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,43 @@
 # Ansible Builder
 
-Ansible Builder is a tool that automates the process of building and shipping a set of execution environments using the schemas and tooling defined in various Ansible Collections.
+Ansible Builder is a tool that automates the process of building execution environments using the schemas and tooling defined in various Ansible Collections and by the user.
+
+## Execution Environments
+
+Execution environments are container images intended to be used by Ansible executors.
+Specifically, `ansible-runner` can make use of these images.
```
Comment (ryanpetrello) [inline README.md]: A link to the Github project URL might be good for `ansible-runner`?

**Review by ryanpetrello — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (beeankha) [reply README.md]: I second that!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,43 @@
...
+
+An execution environment is expected to contain:
+
+ - An install of Ansible (as ansible-base if 2.10 or higher)
```
Comment (beeankha) [inline README.md:12]: Nitpick: add `` around "ansible-base" (so it looks like `ansible-base`)

Comment (beeankha) [inline README.md]: Add the `` around "ansible-base" (so it looks like `ansible-base`)

File: README.md [hunk 3]
```diff
@@ -1,6 +1,43 @@
...
+ - An install of ansible-runner
```
Comment (beeankha) [inline README.md]: Same as above, for `ansible-runner`

File: README.md (hunk 2 above)
Comment (beeankha) [reply README.md]: (and any other instances of `ansible-base`)

**Review by beeankha — APPROVED** [review]
Thank you for doing this, @AlanCoding

**Pull request merged**